.gitignore

```
@@ -1,4 +1,6 @@
 /coverage/
 
+*/.DS_Store
```

I'd recommend writing this as:
```suggestion
.DS_Store
```